src/fragments/lib-v1/storage/js/getting-started.mdx

```diff
@@ -0,0 +1,268 @@
+> Prerequisite: [Install and configure](/cli/start/install) the Amplify CLI
+
+## Storage with Amplify
```

Comment:
Nit. In Auth, you had the first title be "Setup Amplify Auth" but in Storage it's "Storage with Amplify" under "Getting Started". Might worth making it consistent.